webpack.common.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     output: {
         filename: 'server.js',
         path: path.resolve(__dirname, 'server-dist'),
+        chunkFormat: false
```

Comment:
Without this change we get errors like this when running on EC2:
`Error fetching super mode articles from dynamo: Error: Cannot find module './772.server.js`
Though this does not happen locally.

This is quite obscure, but it's due to webpack chunking, see [this thread](https://github.com/aws/aws-sdk-js-v3/issues/5728).
It relates to the switch to [modularized packages](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrating.html) in the v3 sdk. This is a good thing - it reduces the size of AWS dependencies.